### PR TITLE
feat: os notifications

### PR DESCRIPTION
**Pull request by ryanio**
#### What does it do?
* Adds `notify` method to grid and webview APIs for os-level notifications
* Creates `utils/electron.js` file for DRYing `notify` and `openFolderDialog` methods

#### To try it out
Open DevTools and try `Grid.notify('Test Title', 'Test body.')`
* For grid app API: `grid.notify('Test Title', 'Test body.')`

Also try `Grid`/`grid.openFolderDialog()`

#### Future
After this PR is merged I would like to tackle https://github.com/ethereum/grid/issues/274

### CODE REVIEW

**Comment by evertonfraga** [comment]
please put a Please Review label when it's ready! :)

**Comment by PhilippLgh** [comment]
- the new util module needs to be added to files in package.json
- I see a potential problem for certain modules where it is not clear if they are used in main or renderer. so we either decide to make it work in both or make the usage more explicit e.g. renderer-utils 
- the api we are exposing to apps is getting more powerful and potentially dangerous. while it shouldn't be a problem now it could become something we need to restrict e.g. to certain signatures

File: utils/electron.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const { remote } = require('electron')
+const { dialog } = require('electron').remote
+
+const notify = (title, body) => {
+  const notification = new Notification(title, { body })
+  notification.onclick = () => {
+    const window = remote.getCurrentWindow()
```
[the remark this answers is not in the document]
Comment (ryanio) [reply utils/electron.js]: yes currently the way this is coded, whichever window the notification is being triggered from, will be focused on click

**Comment by ryanio** [comment]
> * the new util module needs to be added to files in package.json
👍 done thanks
> * I see a potential problem for certain modules where it is not clear if they are used in main or renderer. so we either decide to make it work in both or make the usage more explicit e.g. renderer-utils
👍agreed, i will make `renderer`  and `main` folders inside `utils`
> * the api we are exposing to apps is getting more powerful and potentially dangerous. while it shouldn't be a problem now it could become something we need to restrict e.g. to certain signatures'
ok, something to definitely keep in mind
> * regarding the notification issue we might be able to trigger it from main process https://electronjs.org/docs/api/notification ?
👍 i have solved this, it turns out localhost is treated as a secure origin so it is working properly

**Pull request merged**